inst/extdata/PA2022CH_en_exec_summary/real-estate-chapter-alternate.Rmd

```diff
@@ -0,0 +1,5 @@
+## Real Estate & Mortgage Results
+
+No Real estate data found
```

Comment:
do we really want an empty page in this case? (I would add one for survey in that case too)
But I find it a bit clunky to then just remove the RE annex and have the annex numbered, `I, II, IV`

Reply:
Checking in with PMs on Teams.